### ISSUE TITLE
I've trained up to 44.1% (test precision)

### ISSUE THREAD

**Issue by carsonDB** [carsonDB/video_tempo#12]
This time, I changed my codes to use multi-gpus to train.
With neural network unchanged.
I have a little different stragety with C3D version (number of iteration of learning rate decay).
Because I get confused by C3D that its paper seems to be different with its configurations
of caffe codes.

And I also met a strange problem. So this network stopped at around 30k iteration.
Unexpected, the clip accuracy is up to 44.1% !!!
Here is the result diagram.
[Tempo_rgb_44.1.tar.gz](https://github.com/carsonDB/Tempo/files/477977/Tempo_rgb_44.1.tar.gz)
